flake8_pep3101.py

```diff
@@ -14,10 +13,7 @@ def __init__(self, tree, filename):
         self.tree = tree
 
     def run(self):
-        if self.filename == 'stdin':
-            lines = pycodestyle.stdin_get_value()
-            tree = ast.parse(lines)
-        elif self.tree:
+        if self.filename is 'stdin':
```

Comment:
@iham this should be ``self.filename == 'stdin'``, python's ``is`` keyword is for identity check:
```
>>> a = 3
>>> b = 3
>>> a is b
True
>>> id(a)
94616584712504
>>> id(b) 
94616584712504
>>> c = 343434343434344
>>> d = 343434343434344
>>> c is d
False
>>> id(c)
94616585066984
>>> id(d)
94616585067032